x-pack/docs/en/rest-api/security.asciidoc

```diff
@@ -71,6 +71,7 @@ without requiring basic authentication:
 * <<security-api-grant-api-key,Grant API key>>
 * <<security-api-query-api-key,Query API key>>
 * <<security-api-update-api-key,Update API key>>
+* <<security-api-bulk-update-api-keys,Bulk update API keys>>
```

Comment:
I'm going for "Bulk update API keys" instead of "Update API keys" for the placeholder, but we can revisit this when we work on the API docs. Bulk is redundant in some sense but it highlights the distinction to "Update API key" for better readability, is explicit, and consistent with the route path `.../_bulk_update`.